docs/src/main/asciidoc/hibernate-search-guide.adoc

```diff
@@ -545,7 +545,7 @@ This is for testing purposes with the Elasticsearch Docker container.
 It should not be used in production.
 
 == Hibernate Search Configuration
-include::generated/quarkus-hibernate-search-elasticsearch.adoc[]
+include::generated/quarkus-hibernate-search-elasticsearch.adoc[leveloffset=+1, opts=optional]
```

Comment:
Oh, I missed you changed the Search guide. I think we should consistently include the configuration reference at the end of the guide with a `Configuration Reference` title.

Considering how long it is, it kinda breaks the guide.

Here, we could just add a note saying `For more information about the Hibernate Search extension configuration, please refer to the link-to-anchor[Configuration Reference]` with a link to an anchor.

Reply:
+1

Reply:
A really good suggestion. Thanks. New changes pushed, WDYT?